packages/services/api/src/modules/token/providers/token-manager.ts

```diff
@@ -82,13 +84,28 @@ export class TokenManager {
     pushIfMissing(scopes, ProjectAccessScope.READ);
     pushIfMissing(scopes, OrganizationAccessScope.READ);
 
-    return this.tokenStorage.createToken({
+    const result = this.tokenStorage.createToken({
       organizationId: input.organizationId,
       projectId: input.projectId,
       targetId: input.targetId,
       name: input.name,
       scopes,
     });
+
+    await this.auditLog.record({
+      eventType: 'TARGET_TOKEN_CREATED',
+      targetId: input.targetId,
+      projectId: input.projectId,
+      alias: input.name,
```

Comment:
TODO @kamilkisiela Should also track the token here? (`result.token`)?

Reply:
Is `result.token` the hashed version of the token?

@dotansimha you know SOC-2 now as good as you know @TuvalSimha, what's your opinion here? What kind of data we need to persist? Should there be an identifier? Is hashed version enough or we could just persist the masked version of the token (a12**********15s)?